subjects/mobile-dev/stock-market/audit/README.md

```diff
@@ -1,30 +1,68 @@
 #### Functional
 
-> In order to run and hot reload app either on emulator or device, follow the [instructions](https://docs.flutter.dev/get-started/test-drive?tab=androidstudio#run-the-app)
+> In order to run and hot reload the app either on emulator or device, follow the [instructions](https://docs.flutter.dev/get-started/test-drive?tab=androidstudio#run-the-app)
 
-###### Does the app run without crushes?
+###### Does the app run without crushing?
 
-###### Signup as new user, and login to your account. Do you have 1000000$ of fake dollars?
+###### Does the app contain a `login/signup` page?
 
-###### After logging in, buy 100 shares of AAPL stock. Check current price of the stock and see if stock appear in your wallet, and your fake dollars are decrease to the according amount.
+##### Signup as new user, and login to your account.
 
-###### Do similar operations with 4 other stocks of your choice. Does the app behave accordingly?
+###### Was is possible to login?
+
+###### Does the app contain a `Wallet` page?
+
+###### Does the app contain a page where you can see the historical data of a stock?
+
+###### Do you have 1000000$ of fake dollars on your account?
```

Comment:
Suggestion - same thing as in the subject about formatting numbers with many digits

Reply:
done